lib/main_publiccloud.pm

```diff
@@ -181,6 +181,7 @@ sub load_publiccloud_tests {
     else {
         loadtest 'boot/boot_to_desktop';
         if (get_var('PUBLIC_CLOUD_MIGRATION')) {
+	    loadtest "publiccloud/upload_image";
```

Comment:
you need to run `make tidy`, or since the code is already commited `make tidy-full`

Reply:
vagrant@localhost:~/os-autoinst-distri-opensuse> make tidy-full
tools/tidy
vagrant@localhost:~/os-autoinst-distri-opensuse> tools/tidy
[checked] lib/main_publiccloud.pm

Reply:
it looks good ..not sure why it fails here..any pointers

Reply:
@yarunachalam https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/3113485273/jobs/5048173760#step:6:69

```
[checked] lib/main_publiccloud.pm
*** needs tidying
```

can you run `make tidy-full`?, before that you should run also `make prepare`... also check that your dependencies are up to date (install `os-autoinst-distri-opensuse-deps` if you're in openSUSE... otherwise ping me over matrix or slack and let's figure out what's wrong with your setup.

if it's a one time thing, align the `loadtest` with the one below it, that should do the tick